tests/test_dirbcast.py

```diff
@@ -78,3 +78,13 @@ def test_DirectedBroadcast(dvs, testlog):
                 assert fvs[0][1] == "FF:FF:FF:FF:FF:FF"
 
     assert dir_bcast
+
+    # Explicitly add a neighbor entry with BCAST MAC and check if its in ASIC_DB
+    dvs.runcmd("ip neigh replace 192.169.0.30 lladdr FF:FF:FF:FF:FF:FF dev Vlan100")
+
```

Comment:
do you need to have a sleep after the command?

Reply:
Yes, it is better. Added now